webgpu.h

```diff
@@ -2173,6 +2755,14 @@ typedef struct WGPUVertexBufferLayout {
     WGPUVertexAttribute const * attributes;
 } WGPUVertexBufferLayout WGPU_STRUCTURE_ATTRIBUTE;
 
+#define WGPU_VERTEX_BUFFER_LAYOUT_INIT _wgpu_MAKE_INIT_STRUCT(WGPUVertexBufferLayout, { \
+    /*.nextInChain=*/NULL _wgpu_COMMA \
+    /*.stepMode=*/WGPUVertexStepMode_Vertex _wgpu_COMMA \
```

Comment:
as noted in the other comment, I think the default here should be `VertexBufferNotUsed`